lib/org/chromium/webidl/CanonicalizerRunner.js

```diff
@@ -32,23 +31,19 @@ foam.CLASS({
       class: 'String',
       name: 'ioRelationshipType',
       documentation: 'The n:m relationship type of input-to-output.',
-      value: 'many:1',
+      value: '*:1',
```

Comment:
Why is it `1:*`? Multiple AST fragments are sent to `CanonicalizerRunner` which will end up producing one `CanonicalCollection`. Thus it is a many-to-one relation, is it not?